liqd_product/assets/scss/components/_headings.scss

```diff
@@ -3,3 +3,7 @@
     text-transform: uppercase;
     letter-spacing: 0.1em;
 }
+
+.heading--sm {
```

Comment:
hmm, this seems to be very specific?! And is not a variant. Ehm, so it either needs another name or might be better to move it to the main_header-component. main-header__brand-something?

Reply:
is small not a modifier?

Reply:
yes, but to use a modifier you have to use both. the class and its modifier